### PR TITLE
Note on terminology and background

### PR DESCRIPTION
**Pull request by Midnighter**

### CODE REVIEW

File: docs/supported_profilers/index.md [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
-The following pages provide more information on the types of output files of each taxonomic profiler can be standardised by `taxpasta`.
+The following pages provide more information on the types of output files of
+each taxonomic profiler can be standardised by taxpasta.
+
+!!! warning
+
+    In particular, when comparing results between various tools, please be very
+    aware of their differences.  In the document on
+    [terminology](terminology.md), we describe how we think about the different
+    tools.  Spending some time on considering the consequences now, will save
```
Comment (jfy133) [inline docs/supported_profilers/index.md:11]:
```suggestion
    tools. Spending some time on considering the consequences now, will save
```

File: docs/supported_profilers/terminology.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+## Criteria
+
+### Type of Abundance/Profiler
+
+Our main criteria for categorizing a metagenomic profiler are summarised in the list below,
+as well as shown in the schema.
+
+-   Reference database:
+    -   **DNA-to-DNA**, aka a metagenome library
+    -   **DNA-to-AA**, aka a genomic library of protein coding sequences
```
Comment (jfy133) [inline docs/supported_profilers/terminology.md:29]:
```suggestion
    -   **DNA-to-AA**, aka a genomic library of protein coding sequences (amino-acid)
```

File: docs/supported_profilers/terminology.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    S --> SP
+    SAA --> SP
+    T --> TP
+```
+
+### Differences in Abundance
+
+A key takeaway from Sun _et al._ (2021)[^1] is also that the different kinds of relative
+abundances cannot easily be interconverted by, for example, normalizing sequence abundance by genome size.
+Reasons for this include, in particular for metagenome-assembled genomes (MAGs), inaccurate
```
Comment (jfy133) [inline docs/supported_profilers/terminology.md:64]:
```suggestion
Reasons for this include (in particular for metagenome-assembled genomes (MAGs)) inaccurate
```

**Review by jfy133 — APPROVED** [review]
I think that's fine for now :) agree we should update over time

File: docs/supported_profilers/terminology.md (hunk 3 above)
Comment (Midnighter) [reply docs/supported_profilers/terminology.md:64]: I prefer less parentheses 😬

Comment (jfy133) [reply docs/supported_profilers/terminology.md:64]: Ok maybe rephrase then - I found it hard to read initially thinking the mags were a part of the list itself not a emphasis
